### PR TITLE
fix for #8365; skip empty lines at the start of a csv file

### PR DESCRIPTION
**Pull request by akbcode**
Small change to parsecsv module

### CODE REVIEW

File: lib/pure/parsecsv.nim [hunk 1]
```diff
@@ -262,4 +264,19 @@ when isMainModule:
 
     # Tidy up
     removeFile("temp.csv")
+  block: # Test for empty line at the beginning of file
+    let content = "\r\nOne,Two,Three,Four\n1,2,3,4\n10,20,30,40,\n100,200,300,400\n"
+    writeFile("temp.csv", content)
```
Comment (Varriount) [inline lib/pure/parsecsv.nim]: Will this work with the tester?

**Comment by Araq** [comment]
Are empty lines ignored in CSV files in general? Why does your CSV start with an empty line?

**Comment by akbcode** [comment]
RFC 4180 does not explicitly allow empty lines. But there's no definitve standard for CSV files. 

I'm processing the output from another tool which puts a blank line at the top for whatever reason.

**Comment by Araq** [comment]
Ok, but this should be fixed differently. The tokenizer should just skip the first newline.

**Pull request closed without merging**